instant-xml-macros/src/lib.rs

```diff
@@ -106,7 +110,7 @@ impl<'a> Serializer {
         };
 
         output.extend(
-            quote!(+ ">" + self.#field_value.to_string().as_str() + "</" + #prefix + #field_name + ">"),
+            quote!(+ ">" + self.#field_value.to_xml(Some(child_prefixes)).unwrap().as_str() + "</" + #prefix + #field_name + ">"),
```

Comment:
I think the use of `+` to concatenate strings isn't very idiomatic, and the performance implications aren't as clear as some of the alternatives. Would be good to do something different here.

Reply:
Changed to `push_str`